### PR TITLE
Fixed faulty Namespace creation.

### PR DESCRIPTION
**Pull request by Michael1993**
Signed-off-by: Mihaly Verhas <misi.verhas@gmail.com>
---

I hereby agree to the terms of the [JUnit Pioneer Contributor License Agreement](https://github.com/junit-pioneer/junit-pioneer/blob/master/CONTRIBUTING.md#junit-pioneer-contributor-license-agreement).

### CODE REVIEW

**Review by Bukama — APPROVED** [review]
Good spot!

**Comment by Bukama** [comment]
As this fixes a namespace which I obviously missed while cleaning up I added this to the milestone.
@aepfli If you find some seconds to verify we could merge this instantly (after providing a short commit message of course :) )

**Review by aepfli — APPROVED** [review]
Lgtm - no time for creating commit message,, so I do not merge for now

**Pull request merged**
